content/docs/iac/concepts/resources/functions.md

```diff
@@ -225,40 +227,54 @@ resources:
 
 {{% /chooser %}}
 
+{{% notes type="info" %}}
+Bridged providers, which take a Terraform provider as an underlying dependency, expose Terraform data sources in the upstream Terraform provider as provider functions in the corresponding Pulumi provider.
+{{% / notes %}}
+
+## Direct form and output form
+
 Provider functions are exposed in each language as regular functions, in two variations:
 
- 1. The **direct form** accepts plain arguments (e.g. `string`, as opposed to `pulumi.Input<string>`) and returns an asynchronous value (e.g. a `Promise` in NodeJS, or a `Task` in Python), or blocks until the result is available. These functions are typically named, e.g., `getFoo()`.
- 2. The **output form** accepts `Input` values (or plain values) and returns an [Output](/docs/concepts/inputs-outputs/). These functions are typically named, e.g., `getFooOutput()`.
+ 1. The **direct form** accepts plain arguments (e.g. `string`, as opposed to `pulumi.Input<string>`) and returns an asynchronous value or blocks until the result is available.
+ 1. The **output form** accepts Pulumi Inputs as arguments and returns a Pulumi as a result. For more information on these types, see [Inputs and Outputs](/docs/concepts/inputs-outputs/).
```

Comment:
```suggestion
 1. The **output form** accepts Pulumi Inputs as arguments and returns a Pulumi Output as a result. For more information on these types, see [Inputs and Outputs](/docs/concepts/inputs-outputs/).
```

Otherwise I'm imaging a purple platypus coming out of the screen.